api/src/main/java/org/openmrs/module/ugandaemrreports/library/Moh105CohortLibrary.java

```diff
@@ -211,4 +211,45 @@ public CohortDefinition hasObsAndEncounter(String encounterType, Concept q, Conc
         cd.setCompositionString("ancEncounter AND hasObs");
         return cd;
     }
+
+	/**
+	 * Number Tetanus Immunizations done
+	 * 
+	 * @return CohortIndicator
+	 */
+	public CohortDefinition tetanusImmunizationsDone(int doseNumber) {
```

Comment:
@moshonk Can you limit this Cohort to the ANC encounter - as this is the only one for pregnant women, and also add the definition for non-pregnant women for future forms... You could just pass a parameter for pregnant